Dangerfile

```diff
@@ -16,7 +16,25 @@ reek.lint
 suggester.suggest
 
 # Report your Ruby app test suite code coverage in Danger.
-simplecov.report 'coverage/coverage.json'
+# simplecov.report 'coverage/coverage.json'
```

Comment:
```suggestion
```

Should we remove this comment about the old way? 🤔

Reply:
Yes, removed in [c60a339](https://github.com/nimblehq/rails-templates/pull/351/commits/c60a339cf9c58a2ab1aca2c0c5011fd4b607cf55)